### PR TITLE
Create an Action of a Type Email

### PR DESCRIPTION
**Pull request by christiansavchev**
The tutorial has been tested in QA and I want to push it to production.

### CODE REVIEW

File: tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+description: Learn how to create an action of a type email
+keywords: tutorial
+auto_validation: true
+time: 10
+tags: [ tutorial>beginner, software-product-function>sap-btp-cockpit ]
+primary_tag: software-product>sap-btp--cloud-foundry-environment
+parser: v2
+---
+
+# Create an Action of a Type Email
```
Comment (MichaelCzcz) [inline tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md:14]: It is already in the meta data. Please remove.

File: tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+<!-- description --> Learn how to create an action of a type email.
```
Comment (MichaelCzcz) [inline tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md:15]: It is already in the meta data. Please remove.

File: tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Prerequisites
+- You have activated Alert Notification in a trial account as described in [Set Up SAP Alert Notification service for SAP BTP for Initial Use](set-up-alert-notification).
+
+## You will learn
+- How to create an action of a type email, so that you can receive notifications to a given email.
```
Comment (MichaelCzcz) [inline tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md:21]: Please remove the full stop

File: tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+3. In the **Actions** dialog box, choose the prefered email action type and then click on the **Next** button.
+
+    ![DialogBox](3-DialogBox.png)
+
+4. Enter a unique name for your action in the **Name** field.
+
+    ![Name](4-Name.png)
+
+5. In the **Email Address** field, enter the destination email to which the notification must be sent and optionally provide a custom template for customizing your notification email. Then click on the **Create** button.
```
Comment (MichaelCzcz) [inline tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md:44]: ...destination email address...

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please address my comments and add at least one validation item to your tutorial!

File: tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md (hunk 1 above)
Comment (jmmargo) [reply tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md:14]: It is better to keep the title and description in the body of the md file and remove it from the meta data.

File: tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md (hunk 4 above)
Comment (jmmargo) [reply tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md:44]: Michael, one would not be able to tell from the md file if there is a validation as in v2, the vr file contains the reference to the step in the md file.  There might not be any indication of a validation in the md file and that is okay.

Comment (MichaelCzcz) [reply tutorials/create-an-action-of-a-type-email/create-an-action-of-a-type-email.md:44]: You are right. I still have to get used to this part ;-)

**Review by christiansavchev — COMMENTED** [review]
Fixed some minor issues.

**Pull request merged**